setup.cfg

```diff
@@ -11,10 +11,14 @@ url = https://potassco.org/
 
 [options]
 packages = find:
-zip_safe = True
+package_dir =
+    =src
 include_package_data = True
 install_requires =
```

Comment:
The template uses `pkg_resources` so `setuptools` is actually a hard requirement. I don't know if listing it in `pyproject.toml` is sufficient.

We should probably switch to `import_metadata`. See <https://setuptools.pypa.io/en/latest/pkg_resources.html>. Currently, the template explicitly supports Python 3.7. I always try to support Python versions that are still maintained (and even a bit beyond that). There also seems to be a backport of the library.

Reply:
I've tested this, and it's sufficient to `setuptools` it in `pyproject.toml`, `pkg_resources` is importable, and the version number shows up on the command line as expected. But since it's deprecated I can put together a commit switching to using `importlib.metadata` / `importlib_metadata` (depending on the python version) to check the version number.